### PR TITLE
[sub-cookbooks] Move install / config to separate cookbooks

### PR DESCRIPTION
**Pull request by charlesg3**
*Issue #, if available:*

*Description of changes:*

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: cookbooks/aws-parallelcluster-install/metadata.rb [hunk 1]
```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+
+name 'aws-parallelcluster-install'
+maintainer 'Amazon Web Services'
+maintainer_email ''
+license 'Apache-2.0'
+description 'Installs/Configures AWS ParallelCluster'
```
Comment (lukeseawalker) [inline cookbooks/aws-parallelcluster-install/metadata.rb:7]: just installs?

File: cookbooks/aws-parallelcluster-config/metadata.rb [hunk 2]
```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+
+name 'aws-parallelcluster-config'
+maintainer 'Amazon Web Services'
+maintainer_email ''
+license 'Apache-2.0'
+description 'Installs/Configures AWS ParallelCluster'
```
Comment (lukeseawalker) [inline cookbooks/aws-parallelcluster-config/metadata.rb:7]: just configures?

File: cookbooks/aws-parallelcluster-install/recipes/install.rb [hunk 3]
```diff
@@ -0,0 +1,31 @@
+# frozen_string_literal: true
+
+#
+# Cookbook Name:: aws-parallelcluster
+# Recipe:: default
```
Comment (lukeseawalker) [inline cookbooks/aws-parallelcluster-install/recipes/install.rb:5]: mismatch between recipe name and file name

File: metadata.rb [hunk 4]
```diff
@@ -24,3 +24,5 @@
 depends 'selinux', '~> 3.1.1'
 depends 'yum', '~> 6.1.1'
 depends 'yum-epel', '~> 4.1.2'
+depends 'aws-parallelcluster-install', '~> 3.0.0'
```
Comment (lukeseawalker) [inline metadata.rb:27]: remember to change the util to bump the cookbook version, to bump aws-parallelcluster-install and aws-parallelcluster-config as well

Comment (charlesg3) [reply metadata.rb:27]: I think I got this -- might help to have a spot-check on it though.

File: util/bump-version.sh [hunk 5]
```diff
@@ -19,6 +19,12 @@ sed -i "s/default\['cluster'\]\['parallelcluster-version'\] = '${CURRENT_PCLUSTE
 sed -i "s/default\['cluster'\]\['parallelcluster-cookbook-version'\] = '$CURRENT_PCLUSTER_VERSION'/default['cluster']['parallelcluster-cookbook-version'] = '${NEW_PCLUSTER_VERSION}'/g" attributes/default.rb
 sed -i "s/default\['cluster'\]\['parallelcluster-node-version'\] = '${CURRENT_PCLUSTER_VERSION}'/default['cluster']['parallelcluster-node-version'] = '${NEW_PCLUSTER_VERSION}'/g" attributes/default.rb
 sed -i "s/version '${CURRENT_PCLUSTER_VERSION_SHORT}'/version '${NEW_PCLUSTER_VERSION_SHORT}'/g" metadata.rb
+sed -i "s/depends 'aws-parallelcluster-install', '~> ${CURRENT_PCLUSTER_VERSION_SHORT}'/depends 'aws-parallelcluster-install', '~> ${NEW_PCLUSTER_VERSION_SHORT}'/g" metadata.rb
+sed -i "s/depends 'aws-parallelcluster-install', '~> ${CURRENT_PCLUSTER_VERSION_SHORT}'/depends 'aws-parallelcluster-config', '~> ${NEW_PCLUSTER_VERSION_SHORT}'/g" metadata.rb
```
Comment (lukeseawalker) [inline util/bump-version.sh:23]: `s/depends 'aws-parallelcluster-config'`

File: util/bump-version.sh [hunk 6]
```diff
@@ -19,6 +19,12 @@ sed -i "s/default\['cluster'\]\['parallelcluster-version'\] = '${CURRENT_PCLUSTE
...
+sed -i "s/depends 'aws-parallelcluster-install', '~> ${CURRENT_PCLUSTER_VERSION_SHORT}'/depends 'aws-parallelcluster-test', '~> ${NEW_PCLUSTER_VERSION_SHORT}'/g" metadata.rb
```
Comment (lukeseawalker) [inline util/bump-version.sh:24]: `s/depends 'aws-parallelcluster-test'`

**Pull request merged**